frontend/src/modules/details/api.ts

```diff
@@ -9,11 +9,16 @@ const fieldsParams = {
   format: 'geojson',
 };
 
+const fieldsParamsDetails = {
+  fields: `${fieldsParams.fields},ratings,ratings_description`,
+  format: 'geojson',
+};
+
```

Comment:
The `fieldsParams` constant doesn't seem to be called anywhere else, why not just add `ratings,ratings_description` to the `fieldsParams` queue?

Reply:
Yes you're right it's better to add it to the fieldParams queue. Thanks for ur advices :) I fix that !